.travis.yml

```diff
@@ -4,9 +4,16 @@ node_js: "10"
 install:
   - npm ci
 
+addons:
+  sonarcloud:
+    organization: ciklum-digital
+    token:
+      secure: $SONAR_TOKEN
+
 script:
   - npm run lint
   - npm run test:ci
+  - sonar-scanner
```

Comment:
Could we move this command into npm scripts
It can be "sonar:ci" : "sonar-scanner"

Reply:
no, because it is the external command

Reply:
but this approach work only for js projects, for other languages we should run sonar-scanner directly